ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -97,31 +121,45 @@ export class InstallationProgress extends React.Component {
         }
 
         return (
-            <Bullseye className={idPrefix + "-status-" + status}>
-                <EmptyStatePanel
-                  icon={icon}
-                  loading={!icon}
-                  paragraph={
-                      status !== "success"
-                          ? <Progress
-                              id={idPrefix + "-bar"}
-                              label={cockpit.format("$0 of $1", step, steps)}
-                              max={steps}
-                              min={0}
-                              title={message}
-                              value={step}
-                              valueText={cockpit.format("$0 of $1", step, steps)}
-                              variant={status}
-                          />
-                          : <Text>{_("Fedora is successfully installed. To begin using it, reboot your system.")}</Text>
-                  }
-                  secondary={
-                      status === "success" &&
-                      <Button onClick={exitGui}>{_("Reboot")}</Button>
-                  }
-                  title={title}
-                />
-            </Bullseye>
+            <EmptyStatePanel
+              icon={icon}
+              loading={!icon}
+              paragraph={
+                  <Flex direction={{ default: "column" }}>
+                      <ProgressStepper isCenterAligned>
+                          {progressSteps.map((progressStep, index) => {
+                              let variant = "pending";
+                              const ariaLabel = _("pending step");
+                              if (index < currentProgressStep) {
+                                  variant = "success";
+                              } else if (index === currentProgressStep) {
+                                  variant = "info";
+                              }
+                              return (
+                                  <ProgressStep
+                                    aria-label={ariaLabel}
+                                    id={idPrefix + "-progress-step-" + index}
```

Comment:
The idPrefix is installation-progress so the final here would be, installation-progress-progress-step-
You can remove the progress from the middle string.

Reply:
Oh, right, fixed. :)